dts/bindings/gpio/nordic,gpio-pins.yaml

```diff
@@ -2,13 +2,13 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
 description: |
-  This allows to define the pin of the GPIO to be owned by the domain.
-  It does not provide any specyfic driver.
-  Its function is to allow us to use two functionalities:
+  This binding allows to match a GPIO to the domain.
+  It does not provide any specific driver.
+  It enables the following functionalities:
 
-  1. Generate the right UICR marking the domain to own the selected pin.
-  2. Access to pins by the macros from zephyr/devicetree/gpio.h file
-     increasing the hardware abstraction this way.
+  - Generate the right UICR marking the domain to own the selected pin.
+  - Access to pins by the macros from the `zephyr/devicetree/gpio.h` file,
```

Comment:
This still sounds a bit odd. I mean "Generate ..." as one of the functionalities.